e2e/definitions/annotated-subject/pv.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: subject-pv
```

Comment:
The PV and PVC look very similar to the other subject.
I think it makes sense to create separate definition dir for them? and include via kustomization.yaml?